graspologic/version/version.txt

```diff
@@ -1 +1 @@
-
+0.1.0.dev20201009170800
```

Comment:
Don't commit any changes to this file - the cleanup is easy (just delete the line and commit it as it is).

Reply:
Would it make sense to put this file into gitignore? Or will that break something

Reply:
I honestly don't know @j1c, but that seems like a good idea going forward. @bdpedigo thoughts?

Reply:
The better fix is to not have the file and rely on its non-existence as the signal that we need to generate one, to be honest.

Reply:
moving this convo to https://github.com/microsoft/graspologic/issues/554 for now just remove any changes you made to this file